laser_encoders/download_models.py

```diff
@@ -71,7 +71,7 @@ def download(self, filename: str):
     def get_language_code(self, language_list: dict, lang: str) -> str:
         try:
             lang_3_4 = language_list[lang]
-            if isinstance(lang_3_4, tuple):
+            if isinstance(lang_3_4, list):
                 options = ", ".join(f"'{opt}'" for opt in lang_3_4)
                 raise ValueError(
                     f"Language '{lang_3_4}' has multiple options: {options}. Please specify using --lang."
```

Comment:
Right now, I am getting the following error message: 
```
ValueError: Language '['kas_Arab', 'kas_Deva']' has multiple options: 'kas_Arab', 'kas_Deva'. Please specify using --lang.
```
I don't like two things about it:
1. I would expect the first part to look like `Language 'kas' has multiple options: 'kas_Arab', 'kas_Deva'. ` , so please replace the first occurrence of `lang_3_4` with `lang` in this error message. 
2. The part `Please specify using --lang.` looks obscure if we use the Python interface instead of CLI. So please rephrase it as  `Please specify using the 'lang' argument.`

Reply:
Thanks @avidale, made the changes,

Reply:
Thanks, this part was written beforehand and I was not sure to change it.
But I have made the changes.